app/helpers/tasklist_helper.rb

```diff
@@ -43,6 +60,10 @@ def review_log_text(log)
 
 private
 
+  def breadcrumb_organisation(log)
+    log.owning_organisation || (log.managing_organisation if log.respond_to?(:managing_organisation))
+  end
+
```

Comment:
So if we navigate from all logs we still want to show the owning org in the breadcrumbs?